api/README.md

```diff
@@ -1,37 +1,13 @@
-| STORAGE_SERVICE_PORT          | no       | 8090          | Port of connected storage service                                                                                                                                                                                                                                                                                                                                                  |
-| STORAGE_SERVICE_EXTERNAL_URL  | no       | -             | IP and port of own connected storage service accessible externally                                                                                                                                                                                                                                                                                                                 |
-| ENCRYPTION_PASSWORD           | no       | -             | If set, all data that is send to the MultiChain node and external storage will be symmetrically encrypted by the ENCRYPTION_PASSWORD                                                                                                                                                                                                                                               |
-| SIGNING_METHOD                | no       | node          | Possible signing methods are: `node` and `user`. Transactions on the chain will be signed using either the address of the node or the address of the specific user publishing that transaction.                                                                                                                                                                                    |
-| NODE_ENV                      | no       | production    | If set to `development` api will allow any string as password. If set to `production` passwords must satisfy safePasswordSchema, see lib/joiValidation-.ts & -.spec.ts files                                                                                                                                                                                                       |
-| ACCESS_CONTROL_ALLOW_ORIGIN   | no       | "\*"          | Since the service uses CORS, the domain by which it can be called needs to be set. Setting this value to `"*"` means that it can be called from any domain. Read more about this topic [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS).                                                                                                                             |
-
 ## Setup
 
 ### Development
 
 Check out our [the ADRs](../doc/adr/) to learn about our current way of doing things.
 
+For additional information, you can checkout the README file under source folder: [README](./src/README.md)
+
```

Comment:
the "Get started" section doesn't mention the fact that we need env vars to start the api. (also no command like `cp .env_example .env`), maybe add that too?